core/arch/aarch64/instr_create.h

```diff
@@ -445,6 +435,16 @@
 #define INSTR_CREATE_bl(dc, pc) \
   instr_create_1dst_1src((dc), OP_bl, opnd_create_reg(DR_REG_X30), (pc))
 
+#define INSTR_CREATE_adc(dc, Rd, Rn, Rm) \
+  instr_create_1dst_2src((dc), OP_adc, (Rd), (Rn), (Rm))
+#define INSTR_CREATE_adcs(dc, Rd, Rn, Rm) \
+  instr_create_1dst_2src((dc), OP_adcs, (Rd), (Rn), (Rm))
+#define INSTR_CREATE_adds_ext(dc, Rd, Rn, Rm, shift, imm3) \
```

Comment:
_extend is already being used (for add, sub, subs) -- seems best to be consistent with that.